backend/app/usecase/requester/verifier.go

```diff
@@ -1,22 +1,27 @@
 package requester
 
-// Verifier verifies in coming network to prevents cyber attacks.
-type Verifier struct {
+// Verifier verifies incoming network to prevent cyber attacks.
+type Verifier interface {
+	IsHuman(recaptchaResponse string) (bool, error)
+}
+
+// VerifierReCaptcha verifies incoming network using ReCaptcha to prevent cyber attacks.
+type VerifierReCaptcha struct {
```

Comment:
We have to make sure that VerifierReCaptcha satisfies Verifier interface. It can be done simply by:
```
var _  Verifier = (*VerifierReCaptcha)(nil)
```

Also in my opinion interface and implementation should be separated. Please look at usecase/repository where are most of the interfaces with fake implementation.

Reply:
Thanks for the feedback! Just made the fixes and pushed, wasn't sure where to put the ReCaptcha implementation of `Verifier` so I left it in the same package as the interface (requester).